README.md

```diff
@@ -77,6 +87,13 @@ main() {
   print("Default= ${inputU.toLowerCase()}, "
       "Turkish=${turkish.toLowerCase(inputU)}\n");
 
+  // compareToTr
+  var zonguldak = "Zonguldak";
+  var cankiri = "Çankırı";
+  print("Comparison for [$zonguldak] and [$cankiri]");
+  print("Default= ${zonguldak.compareTo(cankiri)}, "
```

Comment:
Can you fix the print statements for all `=` symbols? Add a space before and after `=`.